src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IIsPatternExpression.cs

```diff
@@ -0,0 +1,480 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
```

Comment:
>IOperationTests [](start = 25, length = 15)

It would be good to add tests for field initializers, constructor initializers and attribute arguments.